### PR TITLE
LTD-1368 Exporter - page 1

### PR DESCRIPTION
**Pull request by Pithikos**
Frontend work for https://github.com/uktrade/lite-api/pull/857

### CODE REVIEW

File: core/parsing.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+def parse_boolean(value):
```
Comment (eadpearce) [inline core/parsing.py:1]: could this live in core/helpers.py?

Comment (eadpearce) [reply core/parsing.py:1]: actually, if you tweak the form slightly this function won't be necessary

File: exporter/applications/views/documents.py [hunk 2]
```diff
@@ -90,12 +91,70 @@ def document_switch(path):
...
+    return Form(
+        title=strings.TITLE,
+        description=strings.DESCRIPTION,
+        questions=[
+            FileUpload(optional=is_optional),
+            TextArea(title=strings.DESCRIPTION_FIELD_TITLE, optional=True, name="description"),
+            RadioButtons(
+                name="is_content_english",
+                title=strings.Q1_TEXT,
+                options=[Option(key="true", value="Yes"), Option(key="false", value="No"),],
```
Comment (eadpearce) [inline exporter/applications/views/documents.py:125]: change these Options to `Option(key=False, value="No")` and `Option(key=True, value="Yes")` and you won't need to convert the strings "true"/"false" to bools when you get the post data from the form

File: exporter/applications/views/documents.py [hunk 3]
```diff
@@ -26,12 +25,14 @@
...
-from lite_forms.generators import form_page, error_page
-
+from lite_forms.generators import form_page, error_page, confirm_form
+from lite_content.lite_exporter_frontend.applications import DeletePartyDocumentForm
+from lite_forms.components import Form, FileUpload, Label, TextArea, BackLink, Option, RadioButtons
 from core.auth.views import LoginRequiredMixin
 
 
 def document_switch(path):
+    # TODO: Let's remove this turd by creating separate views instead.
```
Comment (eadpearce) [inline exporter/applications/views/documents.py:35]: fix wording pls. this is a public codebase

**Review by eadpearce — CHANGES_REQUESTED** [review]

**Review by eadpearce — APPROVED** [review]

**Comment by Pithikos** [comment]
E2E tests pass locally.

**Comment by saruniitr** [comment]
Please squash commits

**Pull request closed without merging**